shared/text/api/v2.0/auth/oss/token-create.sh

```diff
@@ -0,0 +1,34 @@
+INFLUX_ORG_ID=48c88459ee424a04
```

Comment:
Let's make sure this isn't a real org ID.

Reply:
local